datahub/company/serializers.py

```diff
@@ -286,6 +287,88 @@ def __init__(self, *args, **kwargs):
             for field in self.Meta.dnb_read_only_fields:
                 self.fields[field].read_only = True
 
+    def update(self, instance, validated_data):
+        """
+        Using writable nested representations to copy export country elements
+        from the `Company` model into the `CompanyExportCountry`.
+        """
+        export_to_countries = validated_data.get('export_to_countries', None)
```

Comment:
`None` is the default, so doesn't need to be passed explicitly:

```suggestion
        export_to_countries = validated_data.get('export_to_countries')
```